controllers/apply-next/views/dashboard.njk

```diff
@@ -1,7 +1,134 @@
-{% from "components/application-card/macro.njk" import applicationCard with context %}
+{% extends "layouts/main.njk" %}
 {% from "components/content-box/macro.njk" import contentBox %}
+{% from "components/icons.njk" import iconTick, iconBin %}
 
-{% extends "layouts/main.njk" %}
+{% macro statusMessage(status) %}
+    {% if status === 'ineligible' %}
+        Ineligible
+    {% elseif status === 'complete' %}
+        Submitted for consideration
+    {% else %}
+        Not submitted
+    {% endif %}
+{% endmacro %}
+
+{% macro applicationOverview(application) %}
+    <dl class="o-definition-list o-definition-list--compact u-margin-top">
+        <dt>Proposed start date</dt>
+        <dd>
+            {% set dateParts = application.application_data['project-start-date'] %}
+            {% if dateParts.day and dateParts.month and dateParts.year %}
+                {{ dateParts.day }}/{{ dateParts.month }}/{{ dateParts.year }}
+            {% else %}
+                <span class="application-data-missing">not yet completed</span>
+            {% endif %}
+        </dd>
+        <dt>Grant amount</dt>
+        <dd>
+            {% if application.grantAmount %}
+                £{{ application.grantAmount }}
+            {% else %}
+                <span class="application-data-missing">not yet completed</span>
+            {% endif %}
+        </dd>
+        <dt>Organisation</dt>
+        <dd>
+            {% if application.application_data['organisation-legal-name'] %}
+                {{ application.application_data['organisation-legal-name'] }}
+            {% else %}
+                <span class="application-data-missing">not yet completed</span>
+            {% endif %}
+        </dd>
+        <dt>Project location</dt>
+        <dd>
+            {% if application.application_data['project-postcode'] %}
+                {{ application.application_data['project-postcode'] }}
+            {% else %}
+                <span class="application-data-missing">not yet completed</span>
+            {% endif %}
+        </dd>
+    </dl>
+{% endmacro %}
+
+{% macro applicationCard(application) %}
```

Comment:
I've inlined this back in to this template rather than living as a component file for now. These cards rely on quite a lot of state information and context data and is only used on this page so made sense to keep it in here for now. We can always extract it out again afterwards if it makes sense in future.